src/services/role/index.js

```diff
@@ -20,7 +21,7 @@ module.exports = function setup() {
 		},
 		lean: { virtuals: true },
 		multi: true,
-		whitelist: ['$exists', '$elemMatch', '$regex', '$skip', '$populate'],
+		whitelist: defaultWhitelist,
```

Comment:
```suggestion
		whitelist: [...defaultWhitelist, ...regexWhitelist],
```
usage in https://github.com/hpi-schul-cloud/schulcloud-client/blob/develop/controllers/teams.js#L392